flash/core/utilities/imports.py

```diff
@@ -98,6 +98,7 @@ def _compare_version(package: str, op, version) -> bool:
 _DATASETS_AVAILABLE = _module_available("datasets")
 _ICEVISION_AVAILABLE = _module_available("icevision")
 _ICEDATA_AVAILABLE = _module_available("icedata")
+_LEARN2LEARN_AVAILABLE = _module_available("learn2learn")
```

Comment:
Check the specific version of l2l as I had to update to latest in order to run locally

Reply:
Done !